src/util/mapbox.js

```diff
@@ -110,13 +110,15 @@ export class RequestManager {
 
         const urlObject = parseUrl(tileURL);
         const imageExtensionRe = /(\.(png|jpg)\d*)(?=$)/;
+        const tileURLAPIPrefixRe = /^.+\/v4\//;
```

Comment:
Can you add a comment here that explains why this is necessary?

Reply:
https://github.com/mapbox/mapbox-gl-js/pull/8466#issuecomment-512837197